### DISCUSSION TITLE
please, help  Migration from trinity to firefly

### DISCUSSION THREAD

**Discussion by Denis5328** [iotaledger/firefly#8012] [Help & Support]
I just discovered that I needed to migrate from Trinity to Firefly
I can’t do this in the new version Firefly  there is no seed phrase and in the old version it says network error
Thanks

**Comment by begonaalvarezd** [comment]
The current ability to auto-migrate tokens from the pre-Chrysalis IOTA network via the Firefly wallet have been stopped with the IOTA Stardust upgrade. We will provide a migration tool to resume legacy migrations soon, but its not finished yet. 

Will close the issue because Firefly is not intended to provide migration anymore, but it will be a different tool. The Foundation will notify it when the tool is ready 🌷

**Reply by Denis5328** [reply]
Please tell me when it will be or where I can look for news about it
danke
